vcsutils/webhookparser/gitlab.go

```diff
@@ -148,6 +150,24 @@ func (webhook *gitLabWebhookParser) parsePrEvents(event *gitlab.MergeEvent) (*We
 		TargetBranch:            event.ObjectAttributes.TargetBranch,
 		Timestamp:               eventTime.UTC().Unix(),
 		Event:                   webhookEvent,
+		PullRequest: &PullRequestInfo{
+			ID:        event.ObjectAttributes.IID,
+			Title:     event.ObjectAttributes.Title,
+			Href:      event.ObjectAttributes.URL,
+			Timestamp: eventTime.Unix(),
+			Author: WebHookInfoUser{
+				Login:       event.User.Username,
+				DisplayName: event.User.Name,
+				Email:       event.User.Email,
+				AvatarUrl:   event.User.AvatarURL,
+			},
+			TargetRepository: webhook.parseRepoDetails(event.ObjectAttributes.Target.PathWithNamespace),
+			TargetBranch:     event.ObjectAttributes.TargetBranch,
+			TargetHash:       "",
```

Comment:
Can we get this value from `object_attributes.last_commit`?

Reply:
`object_attributes.last_commit` contains commit SHA of the last commit from the source branch. I have added it to the `SourceHash`.